kernel/h1/src/hil/flash/driver.rs

```diff
@@ -63,13 +63,26 @@ impl<'d, A: Alarm<'d>, H: Hardware> FlashImpl<'d, A, H> {
 }
 
 const MAX_WRITE_SIZE: usize = 32; // Maximum single write is 32 words
+const WORDS_PER_BANK: usize = 0x10000; // 64ki words per bank
+
+// Computes the flash Bank for the specified target location in words
+// from the beginning of flash.
+fn get_bank_from_target(target: usize) -> super::hardware::Bank {
+    if target < WORDS_PER_BANK {
+        super::hardware::Bank::Zero
+    } else {
+        super::hardware::Bank::One
```

Comment:
Should we assert `target < 2*WORDS_PER_BANK` here?

Reply:
The latest commits handle this correctly.

Reply:
Yep -- tx.